recipes/libradolan/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+src_dir="$(pwd)"
```

Comment:
You can use `$SRC_DIR`. Check all `conda-build`'s environment variables [here](http://conda.pydata.org/docs/building/environment-vars.html). They are very in handy sometimes.